andela/travel_tool_front
# #161582665 Mobile responsive for trip planner page

#### What does this PR do?
- add mobile responsive for trip planner page

#### Description of Task to be completed?

- A user should be able to view a web-based mobile version of Travela so that I can have the good UX using my mobile device

#### How should this be manually tested?
- Setting up the backend
     - Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
     - Enter into the project directory - `cd travel_tool_back`
     - checkout the branch git checkout `develop`
     - Install dependencies using `yarn install`
     - Rollback and run database migrations with `yarn db:rollmigrate`


- Setting up the frontend
   - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
   - Enter into the project directory - `cd travel_tool_front`
   - Checkout to the branch `ft-responsive-trip-planer-161582665` then run the app
   - login to the application
   - navigate to the trip planner page as a travel admin
   - right click and select inspect element
   - click on the toggle device toolbar Icon on the bottom left of the screen.
   - choose the device and check the responsiveness of the page.


#### Any background context you want to provide?
I have relied on the browser's simulation of a mobile device's width and have not used actual mobile devices.

#### What are the relevant pivotal tracker stories?

[#161582665](https://www.pivotaltracker.com/story/show/161582665)

#### Screenshots (if appropriate)
![mobiler-responsive](https://user-images.githubusercontent.com/21138053/48214072-380d6d80-e390-11e8-8e1d-99fffb65351f.gif)
